dev-qt/qtwebengine/qtwebengine-5.7.0.ebuild

```diff
@@ -68,14 +68,17 @@ DEPEND="${RDEPEND}
 	dev-util/ninja
 	dev-util/re2c
 	sys-devel/bison
+	pax_kernel? ( sys-apps/elfix )
 "
 
 PATCHES=(
 	"${FILESDIR}/${PN}-5.7.0-icu58.patch"
-	"${FILESDIR}/${PN}-5.7.0-unset-madv_free.patch"
+	"${FILESDIR}/${PN}-5.7.0-undef-madv_free.patch"
```

Comment:
do this in the previous commit, i.e. when you introduce the patch in the first place